viewer/packages/camera-manager/src/Flexible/FlexibleCameraMarkers.ts

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { Scene, Object3D, Vector3, Sprite, SpriteMaterial, CanvasTexture } from 'three';
+import { FlexibleCameraManager } from './FlexibleCameraManager';
+
+const RELATIVE_MARKER_SIZE = 0.02;
+const OUTER_COLOR = '#FF2222';
+const INNER_COLOR = '#FFFFFF';
+
+export class FlexibleCameraMarkers {
```

Comment:
The new update works well! Does this align with the design? Not critical if not - we can change it later

Reply:
He just wanted a dot